ocs_ci/utility/utils.py

```diff
@@ -2054,3 +2055,78 @@ def convert_device_size(unformatted_size, units_to_covert_to):
             return absolute_size / 1024
         elif units == 'Mi':
             return absolute_size
+
+
+def post_ocp_workaround():
+    """
+    Workaround for #1777384 - enable container_use_cephfs on RHEL workers
+    Ticket: RHSTOR-787, see more details in the issue: #1151
+    """
+    # Importing here to avoid circular dependancy
+    from ocs_ci.ocs import ocp
+    from ocs_ci.ocs.node import get_typed_worker_nodes
+    log.info("Running WA for ticket: RHSTOR-787")
+    ocp_obj = ocp.OCP()
+    cmd = ['/usr/sbin/setsebool -P container_use_cephfs on']
+    workers = get_typed_worker_nodes(os_id="rhel")
+    for worker in workers:
+        cmd_list = cmd.copy()
+        node = worker.get().get('metadata').get('name')
+        log.info(
+            f"{node} is a RHEL based worker - applying '{cmd_list}'"
+        )
+        # We saw few times there was an issue to spawn debug RHEL pod.
+        # Let's use retry decorator to make sure our CI is more stable.
+        retry(CommandFailed)(ocp_obj.exec_oc_debug_cmd)(
+            node=node, cmd_list=cmd_list
+        )
+    set_registry_to_managed_state()
+    add_stage_cert()
```

Comment:
please move these 2 functions to be called from the deployment method and not from here